extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -223,7 +221,8 @@ define('ext.wikia.adEngine.adContext', [
 		context.opts.isBLBViewportEnabled = isEnabled('wgAdDriverBottomLeaderBoardViewportCountries');
 		context.opts.additionalBLBSizes = isEnabled('wgAdDriverBottomLeaderBoardAdditionalSizesCountries');
 		context.opts.isBLBSingleSizeForUAPEnabled = isEnabled('wgAdDriverSingleBLBSizeForUAPCountries');
-		context.opts.preFooterAndBLBSwitched = isEnabled('wgAdDriverPreFooterAndBLBSwitchedCountries');
```

Comment:
We need this flag for mobile? I am wrong?
https://github.com/Wikia/mobile-wiki/blob/dev/app/mixins/ads.js#L76

Reply:
I forgot to open up this PR
https://github.com/Wikia/mobile-wiki/pull/874